src/QuadForm/Torsion.jl

```diff
@@ -101,7 +101,7 @@ function torsion_quadratic_module(M::ZZLat, N::ZZLat; gens::Union{Nothing, Vecto
 end
 
 @doc raw"""
-    discriminant_group(L::ZZLat) -> TorQuadModule
+    discriminant_group(L::ZZLat, [n::Int]) -> TorQuadModule
```

Comment:
Does this work for any integer `n` or only prime numbers ? Maybe you could add also a piece of documentation to what the function does when one specifies a value for `n`.